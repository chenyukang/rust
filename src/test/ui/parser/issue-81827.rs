// error-pattern: mismatched closing delimiter: `]`

#![crate_name="0"]



fn main() {}

fn r()->i{0|{#[cfg(r(0{]0
