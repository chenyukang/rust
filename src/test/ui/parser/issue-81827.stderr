error: this file contains an unclosed delimiter
  --> $DIR/issue-81827.rs:11:27
   |
LL | fn r()->i{0|{#[cfg(r(0{]0
   |          -  - -   - -     ^
   |          |  | |   | |
   |          |  | |   | unclosed delimiter
   |          |  | |   unclosed delimiter
   |          |  | unclosed delimiter
   |          |  unclosed delimiter
   |          unclosed delimiter

error: this file contains an unclosed delimiter
  --> $DIR/issue-81827.rs:11:27
   |
LL | fn r()->i{0|{#[cfg(r(0{]0
   |          -  - -   - -     ^
   |          |  | |   | |
   |          |  | |   | unclosed delimiter
   |          |  | |   unclosed delimiter
   |          |  | unclosed delimiter
   |          |  unclosed delimiter
   |          unclosed delimiter

error: expected one of `)` or `,`, found `{`
  --> $DIR/issue-81827.rs:11:23
   |
LL | fn r()->i{0|{#[cfg(r(0{]0
   |                       ^ expected one of `)` or `,`

error: aborting due to 3 previous errors

